src/date-range-picker/embedded.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef } from 'react';
+import Calendar, { DayIndex } from './calendar';
+import InternalBox from '../box/internal';
+import SpaceBetween from '../space-between/index.js';
+
+import RelativeRangePicker from './relative-range';
+import ModeSwitcher from './mode-switcher';
+import { useDateRangePicker } from './use-date-range-picker';
+import { DateRangePickerBaseProps } from './interfaces';
+import { fireNonCancelableEvent } from '../internal/events/index.js';
+import { DateRangePickerProps, Focusable } from './interfaces';
+import { getBrowserTimezoneOffset } from './time-offset';
+import { formatValue } from './use-date-range-picker.js';
+
+export interface DateRangePickerDropdownProps extends DateRangePickerBaseProps {
+  startOfWeek: DayIndex;
+  isSingleGrid: boolean;
```

Comment:
Why isSingleGrid is coming as an argument? It is calculated in the `index.tsx` as `useMobile()`, but the same can be done here.

Reply:
Agreed 👍